about.html

```diff
@@ -0,0 +1,278 @@
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1>
+        </div>
+      </section>
+      <section
+        class="
+          past-summit
+          d-flex
+          flex-column
+          justify-content-center
+          align-items-center
+          mb-5
+        "
+      >
+        <h3 class="mt-5">See the past L2 Conferences</h3>
+        <div class="underline">
+          <hr />
+        </div>
+        <p>Take a Look at the last two L2 Conferences in Nigeria</p>
+        <div class="row d-flex justify-content-center">
+          <div
+            class="
+              col-md-5
+              justify-content-center
+              align-items-center
+              d-flex
+              mb-3
+              Conference-2019
+              flex-column
+              mx-1
+              col-10
+            "
+          >
+          <h2>2019</h2>
+          <p class="text-center">L2 Conference at Petra Ministries, Nigeria</p>
+          </div>
+          <div
+            class="
+            mx-1
+              col-md-5
+              justify-content-center
+              align-items-center
+              d-flex
+              flex-column
+              mb-3
+              Conference-2017
+              col-10
+            "
+          >
+            <h2>2017</h2>
+            <p class="text-center">L2 Conference at Petra Ministries, Nigeria</p>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="container-fluid footer-bg d-md-none">
+        <section
+          class="
+            container
+            d-flex
+            flex-column
+            justify-content-center
+            align-items-center
+          "
+        >
+          <div
+            class="
+              partner-header
+              d-flex
+              flex-column
+              justify-content-center
+              align-items-center
+            "
+          >
+            <h4>Partner</h4>
+            <div class="underline">
+              <hr />
+            </div>
+          </div>
+          <ul
+            class="d-flex list-unstyled flex-wrap justify-content-around col-12"
+          >
+            <li>Petra Ministries</li>
+            <li>Mustard Seed Unilab</li>
+            <li>TACSFON UI</li>
+          </ul>
+        </section>
+      </div>
+      <section
+        class="
+          container-flex
+          d-flex
+          justify-content-around
+          footer-lg
+          align-items-center
+        "
+      >
+      <div class="navbar-brand d-none d-md-block"
+          >              <img src="img/icons/idea.png" alt="brand"> 
+          <h1>
+            L2 Global <br> Conference 2021
+          </h1></div>
+        <p class="text-center">2021 Illuminations. some right reserved</p>
+      </section>
+      <p class="text-center">  
+        License and Credits to Cindy Shin: https://www.behance.net/adagio07</p>
+    </footer>
```

Comment:
The footer section is missing a logo on the left

- [ ] Kindly add the logo on the left.
 _see reference below_
![image](https://user-images.githubusercontent.com/67957533/132740736-669eaf39-94f2-4366-9471-ebbcc01ee74c.png)

Reply:
> The footer section is missing a logo on the left
> 
> * [ ]  Kindly add the logo on the left.
>   _see reference below_
>   ![image](https://user-images.githubusercontent.com/67957533/132740736-669eaf39-94f2-4366-9471-ebbcc01ee74c.png)